packages/mui-material/src/ButtonGroup/ButtonGroup.test.js

```diff
@@ -141,12 +143,13 @@ describe('<ButtonGroup />', () => {
     expect(button).to.have.class('MuiButton-disableElevation');
   });
 
-  it('can disable the ripple', () => {
-    const { container } = render(
+  it('can disable the ripple', async () => {
+    const { container, getByRole } = render(
       <ButtonGroup disableRipple>
         <Button TouchRippleProps={{ classes: { root: 'touchRipple' } }}>Hello World</Button>
       </ButtonGroup>,
     );
+    await ripple.startTouch(getByRole('button'));
```

Comment:
We prefer not to use any of the return values of render(. This is because there is never more than one component mounted, so we can simplify the DX.

```suggestion
    await ripple.startTouch(screen.getByRole('button'));
```

Reply:
Their official linter actually [catches](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/prefer-screen-queries.md) this. Integrating it is on the [code infra board](https://github.com/orgs/mui/projects/21/views/2?pane=issue&itemId=66511046), but this rule is violated all over the place.

Reply:
The rest of this file is already using `getByRole` from `render` a lot, I think I'd prefer to keep the file locally consistent.

Reply:
@Janpot Oh, interesting, I forgot testing-library had an eslint plugin. 👍 to move to `screen.` everywhere in the codebase.

I have promoted this code-infra draft issue, to an open issue: https://github.com/mui/mui-public/issues/173.

Reply:
Perfect timing! I didn't integrating the ESLint plugin was on code-infra's radar already:

- https://mui-org.slack.com/archives/C042YB5RB3N/p1720689276453379
- https://github.com/mui/material-ui/pull/42909